test/builder_test.js

```diff
@@ -109,18 +97,18 @@ test('Builder', function (t) {
         var builder = new Builder(tree)
         builder.cleanup();
         builder.build().then(function (hash) {
-          t.equal(false, true, 'should not succeed')
-          t.end();
+          expect(false).to.equal(true, 'should not succeed');
```

Comment:
This could probably use a better pattern but since we're basically just porting the existing code this seems fine for now

Reply:
yeah I'd imagine a bunch of these are going to change anyway